Thread 1: Fatal error: Unexpectedly found nil while implicitly unwrapping an Optional value

sol0vanti opened the pull request:
When user tries to relocate to another vc (any other vc instead of mapViewVC not depending on when) throws an error.

sol0vanti committed:
Update ViewController.swift

sol0vanti committed:
DetailShopingViewController Uploaded

Beaxhem commented on taxiApp/taxiApp/ViewControllers/ShopingDetailViewController.swift:
```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+class ShopingDetailViewController: ViewController {
```
`UIViewController`, not `ViewController`

sol0vanti committed:
Uploaded (Payment Screen)

Calculation of distance between current user location and final destination annotation in km and price in uah.

skxnz commented:
Thanks
